Cargo.toml

```diff
@@ -16,11 +16,10 @@ path = "src/lib.rs"
 
 [dependencies]
 alga = "0.9"
-nalgebra = "0.19"
+nalgebra = "0.22.1"
 rand = "0.7"
-time = "0.1"
 rayon = "1.2"
-lazy_static = "1.4"
+once_cell = "1.4.1"
```

Comment:
What's the rationale for changing from lazy_static to once_cell?
Can we do this in a separate PR?

Reply:
"once_cell also has a Lazy<T> type, build on top of OnceCell which provides the same API as the lazy_static! macro, but without using any macro" and "The API of once_cell is being proposed for inclusion in std" from https://github.com/matklad/once_cell 

could definitely be a different PR

Reply:
Looks like a reasonable change to me.
But let's move this to a separate PR, please.